src/Resources/views/menu-template.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends 'knp_menu.html.twig' %}
+
+{% block label %}
+    {%- set translation_domain = item.extra('translation_domain', 'navigation') -%}
+    {%- set label = item.label -%}
+    {%- if translation_domain is not same as(false) -%}
+        {%- set label = label|trans(item.extra('translation_params', {}), translation_domain) -%}
+    {%- endif -%}
+    {%- if options.allow_safe_labels and item.extra('safe_label', false) %}{{ label|raw }}{% else %}{{ label }}{% endif -%}
```

Comment:
Only from reading the option names, shouldn't it be the other way around? Or is that option more or less to be interpreted as "did someone else sanitize the label? Yeah, let's dump it straight into the DOM".

Reply:
That is 99,9% the exactly same implementation as in the original bundle: https://github.com/KnpLabs/KnpMenuBundle/blob/master/src/Resources/views/menu.html.twig

The only thing changed is the default translation domain.

Reply:
btw: the name is misleading, but it is intended to say "yeah, that label is already safe, you can paste it directly in the DOM".